examples/fundmover/README.md

```diff
@@ -1,36 +1,53 @@
 # Fundmover Demo App
 
   App demonstrates the usage of TwoPC DM with Maria Database ,
-  BankAccount and Wallet class  extends the TwoPCExtResourceCohortPolicy DM and 
+  BankAccount and Wallet class extends the TwoPCExtResourceCohortPolicy DM and 
   Finance class extends the TwoPCCoordinatorPolicy DM. 
 
-#Setup 
-  Follow the Below step to build and configure the Maria DB
+#Setup  
```

Comment:
suggestion: trail space

Reply:
Trailing space is used in markdown file for adding a newline.
https://stackoverflow.com/questions/33191744/how-to-add-new-line-in-markdown-presentation